ocaml/xapi/xapi_host.ml

```diff
@@ -2978,6 +2978,14 @@ let apply_updates ~__context ~self ~hash =
   (* This function runs on master host *)
   Helpers.assert_we_are_master ~__context ;
   Pool_features.assert_enabled ~__context ~f:Features.Updates ;
```

Comment:
If this function only run on master, and apply update to host, then the parameter `self` is very confusing, 
`let apply_updates ~__context ~self ~hash`
should it be `host`?